.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
```

Comment:
We can add another update for `pip`: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#pip-and-pip-compile

To update the ExamplesTest and our CI script, as this PR is doing: https://github.com/seladb/PcapPlusPlus/pull/1283

cc @clementperon